elastic-agent-shipper.yml

```diff
@@ -7,4 +7,18 @@ tls: false
 #log level
 logging.level: debug
 logging.selectors: ["*"]
-logging.to_stderr: true
\ No newline at end of file
+logging.to_stderr: true
+
+queue:
+  test: #There is no actual "test" queue type, remove this later.
+    events: 512
+
+monitoring:
+  enabled: true
+  interval: 5s
+  log: true
+  http:
+    enabled: true
+    host: "localhost"
+    port: 8282
+    name: "queue"
```

Comment:
It's unclear to me what's the purpose of this file, is it used for testing? A short heading comment would help here :)

Reply:
This is the example config. Added a little comment.